os/arch/xtensa/src/esp32/spi_flash/flash_mmap.c

```diff
@@ -145,7 +145,7 @@ esp_err_t IRAM_ATTR spi_flash_mmap(size_t src_addr, size_t size, spi_flash_mmap_
 #ifdef CONFIG_SPIRAM_SUPPORT
 	int *pages = heap_caps_malloc(sizeof(int) * page_count, MALLOC_CAP_INTERNAL);
```

Comment:
In `heap_caps_malloc`, it still allocates the memory through `malloc_at`. We'd better change this also.

Reply:
Done